spec/javascripts/app/views/single-post-view/single_post_comment_stream_spec.js

```diff
@@ -9,10 +9,91 @@ describe("app.views.SinglePostCommentStream", function() {
       expect(this.view.CommentView).toBe(app.views.ExpandedComment);
     });
 
+    it("calls highlightPermalinkComment on hashchange", function() {
```

Comment:
nit: this name is misleading. `initialize` doesn't call `highlightPermalinkComment`, it sets a handler for `hashchange` event.